pkg/platform/packet/packet_test.go

```diff
@@ -355,3 +355,31 @@ func TestTerraformAddDeps(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckWorkerPoolLabelsWithSpacedValue(t *testing.T) {
```

Comment:
We already have `TestCheckValidConfig` which starts with valid configuration and lets you mutate it to introduce a bad config option. I think we should use this, as eventually it will allow only testing using exported interface.